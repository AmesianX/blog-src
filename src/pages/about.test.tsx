import React from 'react'
import { render, cleanup } from '@testing-library/react'

import AboutPage from './about'

jest.mock('hooks/use-site-metadata')

describe('page/AboutPage', () => {
    afterEach(cleanup)

    it('should render properly', () => {
        const { container } = render(<AboutPage/>)
        expect(container).toMatchSnapshot()
    })
})
