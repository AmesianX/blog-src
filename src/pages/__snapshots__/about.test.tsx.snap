// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page/AboutPage should render properly 1`] = `
.c0 {
  position: fixed;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0;
  width: 100%;
  height: 3.75rem;
  background: #fff;
  border-bottom: 1px solid #e1e1e1;
  -webkit-transition: -webkit-transform .5s ease;
  -webkit-transition: transform .5s ease;
  transition: transform .5s ease;
  z-index: 2;
}

.c0.hide {
  -webkit-transform: translateY(-3.75rem);
  -ms-transform: translateY(-3.75rem);
  transform: translateY(-3.75rem);
}

.c1 {
  position: absolute;
  left: 1.25rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1.5rem;
  font-weight: bold;
  color: rgba(0,0,0,0.78);
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0;
  width: 100%;
  height: 3rem;
}

.c3 {
  text-align: center;
}

.c3,
.c3 a {
  font-size: .8rem;
  color: darkgray;
}

.c3 + :before {
  content: '·';
  margin: 0 .1rem;
}

.c3 a {
  font-weight: bold;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<div>
  <header
    class="c0"
  >
    <a
      class="c1"
      href="/"
    >
      cometkim's dev blog
    </a>
  </header>
  <footer
    class="c2"
  >
    <section
      class="c3"
    >
      © 2018 
      <a
        href="/about/"
      >
        Hyeseong Kim
      </a>
    </section>
    <section
      class="c3"
    >
      Powered by 
      <a
        href="https://gatsbyjs.org"
        target="_blank"
      >
        GatsbyJS
      </a>
    </section>
    <section
      class="c3"
    >
      Hosted by 
      <a
        href="https://netlify.com"
        target="_blank"
      >
        Netlify
      </a>
    </section>
    <section
      class="c3"
    >
      Source code on 
      <a
        href="https://github.com/cometkim/blog-src"
        target="_blank"
      >
        GitHub
      </a>
    </section>
  </footer>
</div>
`;
